src/set_theory/zfc/basic.lean

```diff
@@ -677,6 +677,31 @@ theorem singleton_injective : function.injective (@singleton Set Set _) :=
 @[simp] theorem to_set_sUnion (x : Set.{u}) : (⋃₀ x).to_set = ⋃₀ (to_set '' x.to_set) :=
 by { ext, simp }
 
+/-- The intersection operator, the collection of elements in all of the elements of a ZFC set. We
+special-case `⋂₀ ∅ = ∅`. -/
+noncomputable def sInter (x : Set) : Set :=
+begin
+  by_cases h : x.nonempty,
+  { exact {y ∈ classical.some h | ∀ z ∈ x, y ∈ z} },
+  { exact ∅ }
+end
+
+prefix (name := Set.sInter) `⋂₀ `:110 := Set.sInter
```

Comment:
Can you check whether this renders the `set.sInter` notation unusable? If so, can you scope this one?

Reply:
This is just copy + replace the `Set.sUnion` notation that already exists in this file, and that hasn't caused any problems.